examples/image_classification/run_all_models.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
```

Comment:
Is it supposed to run setup.sh before running python script? If it was left out by mistake, please add more lines for it.